tests/opera/parser/test_tosca.py

```diff
@@ -101,3 +101,41 @@ def test_load_from_csar_subfolder(self, tmp_path, yaml_text):
 
         doc = tosca.load(tmp_path, name)
         assert doc.data_types["my_type"]
+
+    def test_duplicate_import(self, tmp_path, yaml_text):
+        name = pathlib.PurePath("template.yaml")
+        (tmp_path / name).write_text(yaml_text(
+            """
+            tosca_definitions_version: tosca_simple_yaml_1_3
+            imports: [ template.yaml ]
+            """
+        ))
+        tosca.load(tmp_path, name)
+
+    def test_duplicate_imports(self, yaml_ast, yaml_text, tmp_path):
```

Comment:
`yaml_ast` fixture is not needed in this test. Please remove it.

Reply:
Ok, this was removed.